lusSTR/cli/snps.py

```diff
@@ -23,3 +23,40 @@ def subparser(subparsers):
     p.add_argument("--config", default="config.yaml", help="config file used to identify settings.")
     p.add_argument("-w", "--workdir", metavar="W", default=".", help="working directory")
     p.add_argument("--skip-filter", dest="filter", action = "store_true", help="Skip filtering step")
+
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2020, DHS.
+#
+# This file is part of lusSTR (http://github.com/bioforensics/lusSTR) and is licensed under
+# the BSD license: see LICENSE.txt.
```

Comment:
Was it intentional to paste this below the placeholder above?

Reply:
Ha! I've been scrolled down on this script I've never even realized this was there